airbyte-webapp/package.json

```diff
@@ -66,7 +66,7 @@
     "react-app-rewired": "^2.1.5",
     "react-hot-loader": "^4.12.20",
     "react-scripts": "3.4.3",
-    "typescript": "3.9.7"
+    "typescript": "4.0.3"
```

Comment:
If we are going to bump version up - do you think we can upgrade it to the latest one`4.1.`?

Reply:
oh ya, that should be fine, i was just being conservative :p

Reply:
Updated the pr